README.md

```diff
@@ -53,4 +53,21 @@ Once the container is running, it should be accessible using:
 $ curl 127.0.0.1:5000
 ```
 
-Personally, I use a Mac to generate my code and docker image so I just open a browser to http://192.168.99.100:5000/`
+# Openshift Usage
+
+create the s2i builder image, dont need service or deployment
+
+    oc new-project aspnet --display-name="ASP.NET s2i" --description='ASP.NET s2i'
+    oc new-app https://github.com/eformat/s2i-aspnet --name=aspapp --strategy=docker
+    oc delete svc aspapp
+    oc delete dc aspapp
+
+s2i needs access to docker dameon, so as the cluster amdin
```

Comment:
@jawnsy I don't know.   Now that the dockerfile is running non-root, maybe you don't need cluster admin.   I'll have to test out the scenarios.    Can we accept the change (since we're not in engineering) and I'll fix with a later PR.